chanzuckerberg/single-cell-data-portal
# feat(cellGuide): 5208 SEO synonyms cell type page

## Reason for Change

- #5208
- If the reason for this PR's code changes are not clear in the issue, state value/impact

## Changes

1. Update tests to test synonyms both in UI and SEO meta tags
2. Add SDS design token utility functions to improve code quality and DX (will have a follow up PR today to refactor the whole app with the utility functions)
3. Extract `<GeneSynonyms />` component to be a generic `<Synonyms />` component, so it can be used for both genes and cell types
4. Ignore `html-reports` for stylelint

## Testing steps

1. Go to https://localhost:3000/cellguide/CL_0000185 (Myoepithelial Cell)
2. You should see:
    
<img width="1345" alt="Screenshot 2023-07-26 at 9 31 36 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/6309723/21778616-200b-4ef4-8461-83c5cdb453ed">


## Notes for Reviewer
